recipe/conda_build_config.yaml

```diff
@@ -200,12 +206,12 @@ vc:
   - 14
   - 14
 vlfeat:
-  - 0.9
-xerces_c:
-  - 3
+  - 0.9.20
+xerces-c:
+  - 3.2.0
```

Comment:
This seems really tight, but that is what [we and defaults have]( https://github.com/conda-forge/conda-forge.github.io/blob/61cdd7fd0374c07e95afdcbd414c7985b141e4b3/scripts/pin_the_slow_way.py#L126 ) apparently. Did not find API/ABI info, but did request it in issue ( https://github.com/lvc/upstream-tracker/issues/29 ).

Reply:
Here's the newly added [API/ABI report]( https://abi-laboratory.pro/tracker/timeline/xerces-c/ ). Probably 3.2 is sufficient, but would be good to hear from `xerces-c` users. @conda-forge/xerces-c?